config/ember-try.js

```diff
@@ -3,128 +3,99 @@
 const getChannelURL = require('ember-source-channel-url');
 const { embroiderSafe, embroiderOptimized } = require('@embroider/test-setup');
 
-module.exports = async function() {
+module.exports = async function () {
   return {
     useYarn: true,
     scenarios: [
       {
-        name: 'ember-lts-3.4',
-        npm: {
-          devDependencies: {
-            'ember-source': '~3.4.0'
-          }
-        }
-      },
-      {
-        name: 'ember-lts-3.8',
-        npm: {
-          devDependencies: {
-            'ember-source': '~3.8.0'
-          }
-        }
-      },
-      {
-        name: 'ember-lts-3.12',
+        name: 'ember-lts-3.24',
```

Comment:
@knownasilya can/should we see any thing below, I would guess 3.20, 3.16 should be no brainer?

Reply:
Can use the old version of this addon for those, otherwise I don't see why we should go beyond the current versions. Doesn't mean you can't use with those versions, just no guarantees.

Reply:
These versions come from the ember-cli-update codemod btw.